cellprofiler/__main__.py

```diff
@@ -1,4 +1,14 @@
 from __future__ import print_function
+
+import os
+import os.path
+import sys
+if sys.platform.startswith("win") and hasattr(sys, 'frozen'):
+    # Allow us to package java with windows builds
+    if 'JAVA_HOME' not in os.environ:
+        if 'CP_JAVAHOME' in os.environ:
```

Comment:
This should probably be `CP_JAVA_HOME` for consistency

Reply:
K